File: dotnet/src/MyDotey.SCF/Facade/ConfigurationProperties.cs
using System;

namespace MyDotey.SCF.Facade
{
    /**
     * @author koqizhao
     *
     * May 17, 2018
     */
    public class ConfigurationProperties
    {
        protected ConfigurationProperties()
        {

        }

        public static PropertyConfig<K, V>.IBuilder NewConfigBuilder<K, V>()
        {
            return new DefaultPropertyConfig<K, V>.Builder();
        }
    }
}

File: java/src/main/java/org/mydotey/scf/facade/ConfigurationProperties.java
package org.mydotey.scf.facade;

import org.mydotey.scf.DefaultPropertyConfig;
import org.mydotey.scf.PropertyConfig;

/**
 * @author koqizhao
 *
 * May 17, 2018
 */
public class ConfigurationProperties {

    protected ConfigurationProperties() {

    }

    public static <K, V> PropertyConfig.Builder<K, V> newConfigBuilder() {
        return new DefaultPropertyConfig.Builder<>();
    }

}
